### PR TITLE
Add CozyCal Scheduling to integrations

### PR DESCRIPTION
**Pull request by csytan**
Let me know if I'm missing anything. Thanks!

### CODE REVIEW

**Comment by csytan** [comment]
Hi @intrip 

Thanks for looking into this PR, and sorry for the late reply.

Yes, we have a support page about the integration, but it's pretty short:
https://support.cozycal.com/basecamp-integration

Do you require a landing page for Basecamp users?

File: integrations.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+-
+  name: "CozyCal Scheduling"
+  description: "Allow your guests to book support calls, video meetings, rooms, and more. New events are automatically added your Basecamp schedule."
+  website: "https://www.cozycal.com/?ref=basecamp"
```
Comment (intrip) [inline integrations.yml:17]: can you please change this to the page you linked in https://github.com/basecamp/bc3-integrations/pull/191#issuecomment-1154486375

File: integrations.yml [hunk 2]
```diff
@@ -11,6 +11,12 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+-
+  name: "CozyCal Scheduling"
```
Comment (intrip) [inline integrations.yml:15]: This needs to be appended to the bottom of the .yml file to respect historical ordering

**Comment by intrip** [comment]
Thanks @csytan!

I think this [page](https://support.cozycal.com/basecamp-integration) is ok /cc @jorgemanrubia.
Please check my other comments.

**Comment by csytan** [comment]
Thanks for the feedback @intrip 

I've made the following changes:
- Moved CozyCal to the bottom of the .yml file
- Updated the website to point to a proper landing page (https://www.cozycal.com/basecamp-integration)

**Pull request merged**